conference.js

```diff
@@ -1093,6 +1094,9 @@ export default {
      * @param detailedFeedback detailed feedback from the user. Not yet used
      */
     sendFeedback(overallFeedback, detailedFeedback) {
```

Comment:
Optional: I don't see why this function needs to be in conference.js. It will be nice to move it out of conference.js. 

NOTE: This change is not related to your changes in the PR. But it should pretty easy to move it to react/redux.

Reply:
I don't think this one is actually used anywhere. Think it's confusion of names, because in the feedback we call 'conference' the conference.room from conference.js. Basically conference in Feedback actions is the JitsiConference object from lib-jitsi-meet.

Reply:
Removed this function completely as is does not seem to be used anywhere.